code/threshold_canny.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<p>Given the new insights I got today, I think it would be more important to display the sharp edge of the liver liver in the top right of the image</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/markusait/lcmd/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>